pkg/yurttunnel/pki/certmanager/certmanager.go

```diff
@@ -120,7 +131,9 @@ func newCertManager(
 	certDir,
 	commonName string,
 	organizations,
-	dnsNames []string, ipAddrs []net.IP) (certificate.Manager, error) {
+	dnsNames []string,
+	keyUsage []certificates.KeyUsage,
```

Comment:
I think keyUsage is an plural var, so how about change `keyUsage` to `keyUsages`?

Reply:
fix